app_dart/lib/src/service/luci.dart

```diff
@@ -99,6 +99,17 @@ class LuciService {
     return results;
   }
 
+  @visibleForTesting
+
```

Comment:
nit: newline

Reply:
Do you meant to remove this newline? New line is auto generated via format.
Moved the doc above the annotation.